typesafe/src/test/scala/zio/config/typesafe/TypesafeConfigSpec.scala

```diff
@@ -1,16 +1,30 @@
 package zio.config.typesafe
 
-import zio.test._
-import zio.test.Assertion._
-import zio.config.BaseSpec
-import TypesafeConfigSpecUtils._
 import zio.config.PropertyTree.{ Leaf, Record, Sequence }
-import zio.config._
+import zio.config.{ BaseSpec, _ }
 import zio.config.magnolia.DeriveConfigDescriptor._
+import zio.config.typesafe.TypesafeConfigSpecUtils._
+import zio.test.Assertion._
+import zio.test._
 
 object TypesafeConfigSpec
     extends BaseSpec(
       suite("TypesafeConfig")(
+        test("Read empty list") {
+          val res =
+            TypeSafeConfigSource.fromHoconString(
+              """
+                |a {
+                |  b = "s"
+                |  c = []
+                |}
+                |""".stripMargin
+            )
+
+          val expected = Record(Map("a" -> Record(Map("b" -> Leaf("s"), "c" -> Sequence(Nil)))))
+
+          assert(res.map(_.getConfigValue(List.empty)))(isRight(equalTo(expected)))
```

Comment:
👍

Reply:
Could we change the test just below this one and change some of the values to empty list (at different places) and assert this is true in all cases?

This is fantastic !

Reply:
done